### ISSUE TITLE
undefined method `get_listing' Android beta build

### ISSUE THREAD

**Issue by rogerkerse** [fastlane/fastlane#10148]
### New Issue Checklist

- [2.54.1 ] Updated fastlane to the latest version

### Issue Description

when trying to release a beta build to play store i get this error `undefined method `get_listing' for #<Google::Apis::AndroidpublisherV2::AndroidPublisherService:0x007fa04c12ed98>`

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

```
[14:07:20]: undefined method `get_listing' for #<Google::Apis::AndroidpublisherV2::AndroidPublisherService:0x007fa04c12ed98>

+------+---------------------+-------------+
|             fastlane summary             |
+------+---------------------+-------------+
| Step | Action              | Time (in s) |
+------+---------------------+-------------+
| 1    | Verifying required  | 0           |
|      | fastlane version    |             |
| 2    | default_platform    | 0           |
| 3    | gradle              | 174         |
| 💥   | supply              | 2           |
+------+---------------------+-------------+

[14:07:20]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
/Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/client.rb:145:in `listing_for_language': [!] undefined method `get_listing' for #<Google::Apis::AndroidpublisherV2::AndroidPublisherService:0x007fa04c12ed98> (NoMethodError)
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:17:in `block in perform_upload'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:13:in `each'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:13:in `perform_upload'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/actions/supply.rb:21:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:33:in `block (2 levels) in parsing_binding'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/bin/fastlane:20:in `<top (required)>'
	from /Users/rogerkerse/.fastlane/bin/bundle/bin/fastlane:22:in `load'
	from /Users/rogerkerse/.fastlane/bin/bundle/bin/fastlane:22:in `<main>'
```

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.2.4                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.11.0 (Apple Git-81)           |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | true                                        |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.53.1"

default_platform :android

platform :android do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
  end

  desc "Runs all the tests"
  lane :test do
    gradle(task: "test")
  end

  # desc "Submit a new Beta Build to Crashlytics Beta"
  desc "Submit a new Beta Build to Play Store Beta"
  lane :beta do
    gradle(task: "assembleProductionRelease")
    # crashlytics # If you want to upload it to crashlytics instead of play store
    supply(track: "beta")

    # sh "your_script.sh"
    # You can also use other beta testing services here
  end

  desc "Deploy a new version to the Google Play"
  lane :deploy do
    gradle(task: "assembleProductionRelease")
    supply
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used. No personal data is recorded.
# Learn more at https://github.com/fastlane/fastlane#metrics
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file "app/fastlane-play-store-key.json" # Path to the json secret file - Follow https://github.com/fastlane/supply#setup to get one
package_name "ee.bolt.unispotter" # e.g. com.krausefx.app
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.54.1  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.1        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.55.0       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.12.0       |
| json                      | 1.8.1        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.0        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.11.0.1     |
</details>

*generated on:* **2017-08-25**
</details>

**Comment by mpirri** [comment]
@rogerkerse Looks like you are using the google-api-client version 0.12.0 - any chance you could do a gem update on that gem to get to version 0.13.x? I think it would solve the missing method problem you are seeing. In the meantime, I will look into why it is that the gem didn't automatically update when you installed the latest version of fastlane.

Thanks, please let us know if you have any luck with this! 🚀

**Comment by rogerkerse** [comment]
@mpirri how can I do that when I am using brew version of fastlane

**Comment by rogerkerse** [comment]
Ok managed to get 0.13.x with `brew cask reinstall fastlane`. Will let you know if this fixes the issue, but yeah, with fastlane upgrading the version did not come

**Comment by rogerkerse** [comment]
The issue did not stop there.

New Issue in same place

```
[17:05:34]: undefined method `deleteall_image' for #<Google::Apis::AndroidpublisherV2::AndroidPublisherService:0x007f9aafeb2890>

+------+---------------------+-------------+
|             fastlane summary             |
+------+---------------------+-------------+
| Step | Action              | Time (in s) |
+------+---------------------+-------------+
| 1    | Verifying required  | 0           |
|      | fastlane version    |             |
| 2    | default_platform    | 0           |
| 3    | gradle              | 36          |
| 💥   | supply              | 13          |
+------+---------------------+-------------+

[17:05:34]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
/Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/client.rb:336:in `block in clear_screenshots': [!] undefined method `deleteall_image' for #<Google::Apis::AndroidpublisherV2::AndroidPublisherService:0x007f9aafeb2890> (NoMethodError)
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/client.rb:367:in `call_google_api'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/client.rb:335:in `clear_screenshots'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:96:in `block in upload_screenshots'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:91:in `each'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:91:in `upload_screenshots'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:21:in `block in perform_upload'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:13:in `each'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/supply/lib/supply/uploader.rb:13:in `perform_upload'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/actions/supply.rb:21:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:33:in `block (2 levels) in parsing_binding'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
	from /Users/rogerkerse/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.54.1/bin/fastlane:20:in `<top (required)>'
	from /Users/rogerkerse/.fastlane/bin/bundle/bin/fastlane:22:in `load'
	from /Users/rogerkerse/.fastlane/bin/bundle/bin/fastlane:22:in `<main>'
```

ENV 

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.2.4                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.11.0 (Apple Git-81)           |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | true                                        |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                       |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.53.1"

default_platform :android

platform :android do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
  end

  desc "Runs all the tests"
  lane :test do
    gradle(task: "test")
  end

  # desc "Submit a new Beta Build to Crashlytics Beta"
  desc "Submit a new Beta Build to Play Store Beta"
  lane :beta do
    gradle(task: "assembleProductionRelease")
    # crashlytics # If you want to upload it to crashlytics instead of play store
    supply(track: "beta")

    # sh "your_script.sh"
    # You can also use other beta testing services here
  end

  desc "Deploy a new version to the Google Play"
  lane :deploy do
    gradle(task: "assembleProductionRelease")
    supply
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used. No personal data is recorded.
# Learn more at https://github.com/fastlane/fastlane#metrics
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file "app/fastlane-play-store-key.json" # Path to the json secret file - Follow https://github.com/fastlane/supply#setup to get one
package_name "ee.bolt.unispotter" # e.g. com.krausefx.app
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.54.1  | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.1        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.55.0       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.1       |
| json                      | 1.8.1        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.0        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.11.0.1     |
</details>


*generated on:* **2017-08-25**
</details>

**Comment by rogerkerse** [comment]
Ok as I can see, it will be fixed in some version of fastlane. Any info when it will happen?

**Comment by mpirri** [comment]
I'm going to kick off a release very shortly, so it should be available in the next hour or two. Sorry for the delay on this!

**Comment by mpirri** [comment]
@rogerkerse Okay the new release is available (2.54.2) - give it a try and let us know if it helps.  Thanks again!

**Comment by tmtrademarked** [comment]
There were still a few missed APIs. I've uploaded https://github.com/fastlane/fastlane/pull/10153 which should correct the last ones.
